platform-services-handler/index.js

```diff
@@ -117,25 +117,26 @@ module.exports.handler = (event, context, cb) => {
 
```

Comment:
Make sure all service handler functionalities  from Internal version is ported

Reply:
All the features are not merged from internal version